### DISCUSSION TITLE
No collecting endpoint logs with Elastic Agent

### DISCUSSION THREAD

**Discussion by jrdrey** [Security-Onion-Solutions/securityonion#13144] [2.4] [answered]
### Version

2.4.70

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Eval

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

16

### Storage for /

69,2

### Storage for /nsm

133,7

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

 _![Capture d'écran 2024-06-05 142050](https://github.com/Security-Onion-Solutions/securityonion/assets/66970551/c5064623-db12-4eb8-9d93-09dd77080497)
![Capture d'écran 2024-06-05 154749](https://github.com/Security-Onion-Solutions/securityonion/assets/66970551/9716c537-e078-49f5-9d4c-c9d5c647502b)
![Capture d'écran 2024-06-05 141801](https://github.com/Security-Onion-Solutions/securityonion/assets/66970551/17c16292-386e-484a-bed1-ee1ec9f98bfd)_

after installing the elastic agents on two clients (windows 10 and ubuntu), i notice that the logs don't go back up to the SOC (security onion console) and i've noticed the absence of logstash in the _/opt/so/conf/_ and _/opt/so/log/_ directories and the command: _sudo so-logstash-restart_ returns:

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment, marked as answer]
You specify above that your installation type is Eval (Evaluation mode). Evaluation Mode does not support remote agents. 

If you downloaded your Elastic Agents from SOC Downloads, there should have been a red banner mentioning this:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/1ba2bc41-7f6f-45a2-a5e7-97fc095fb2f8)

This is also mentioned in several locations throughout the documentation. From https://docs.securityonion.net/en/2.4/elastic-agent.html#deployment:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/8f827016-1638-43c5-a037-e20658f3eb1a)

From https://docs.securityonion.net/en/2.4/architecture.html#evaluation:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/5250878f-6824-45af-b0f4-e692a7f224c7)

From https://docs.securityonion.net/en/2.4/downloads.html#downloads:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/1659467/08363d45-3e89-4f26-84fa-743629d7d66b)
